commercetools-models/src/main/java/io/sphere/sdk/shippingmethods/commands/updateactions/ChangeName.java

```diff
@@ -1,28 +0,0 @@
-package io.sphere.sdk.shippingmethods.commands.updateactions;
-
-import io.sphere.sdk.commands.UpdateActionImpl;
-import io.sphere.sdk.shippingmethods.ShippingMethod;
-
-/**
- * Changes the name.
- *
- * {@doc.gen intro}
- *
- * {@include.example io.sphere.sdk.shippingmethods.commands.ShippingMethodUpdateCommandIntegrationTest#changeName()}
- */
-public final class ChangeName extends UpdateActionImpl<ShippingMethod> {
```

Comment:
Great that you replaced the manually written update actions with generated ones 👍